otelcollector/scripts/setup.sh

```diff
@@ -35,16 +35,16 @@ gem install deep_merge
 gem install re2
 
 echo "Installing mdsd..."
-if [ "${ARCH}" != "amd64" ]; then
-  wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_aarch64.rpm
-  sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_aarch64.rpm
-else
-  wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_x86_64.rpm
-  sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_x86_64.rpm
-fi
+# if [ "${ARCH}" != "amd64" ]; then
+#   wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_aarch64.rpm
+#   sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_aarch64.rpm
+# else
+#   wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_x86_64.rpm
+#   sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_x86_64.rpm
+# fi
 
 # Install this way once moving to the Mariner published RPMs:
-# sudo tdnf install -y azure-mdsd
+sudo tdnf install -y azure-mdsd_1.25.1
```

Comment:
Please upgrade the windows MA version too. We will need to verify data flow once the update is done to make sure that MA now has the prometheus stream changes.

Reply:
Sure. Thanks Kaveesh!

Reply:
I have verified that metrics are flowing and there are no error logs in the windows and linux pods.